src/__test__/iotx.integration.test.ts

```diff
@@ -56,11 +60,11 @@ test.skip("transfer", async t => {
     "account must have 1 IOTX"
   );
 
-  const acctNew = antenna.iotx.accounts.create("any entropy");
+  const acctNew = "io13zt8sznez2pf0q0hqdz2hyl938wak2fsjgdeml";
```

Comment:
do not update unrelated test?

Reply:
easy to verify transfer result, already resume previous version